src/appshell/qml/platform/AppMenuBar.qml

```diff
@@ -28,7 +28,7 @@ import MuseScore.AppShell 1.0
 ListView {
     id: root
 
-    height: contentItem.childrenRect.height
+    height: Math.max(1,contentItem.childrenRect.height)
```

Comment:
@kbloom don't we need the same change for sectionContentList in MixerPanelSection.qml? Some linux users have reported that the mixer is not displayed for them with later Qt versions (6.7.2, for example). See the comments https://github.com/musescore/MuseScore/issues/24885

Reply:
Looks like we probably do. But because panels are broken more generally in Qt 6.5+ (#24866), making that change didn't actually fix anything when I tested it.  I can add that to this PR tomorrow, but I won't have time to dig into the Qt stuff that @orivej did in #24866 to test it.